e2e/pollers.go

```diff
@@ -58,14 +59,26 @@ func waitUntilNodeReady(ctx context.Context, t *testing.T, kube *Kubeclient, vms
 	return nodeName
 }
 
-func waitUntilPodReady(ctx context.Context, kube *Kubeclient, podName string) error {
+func waitUntilPodReady(ctx context.Context, kube *Kubeclient, podName string, t *testing.T) error {
```

Comment:
I think adding some logs here can help determine if your e2e test setup is going to fail without having to wait the 15 minutes for the timeout. And if everything is working, it doesn't produce that many more log lines (like 4-6 I believe)